### ISSUE TITLE
Preferences - Disable Material Icon Rendering is unstable with 300+ materials

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#4824]
![image](https://github.com/user-attachments/assets/df0b96dd-b475-4473-b714-1b1f31c85c81)

Been trying out and last night with a library from work with over 300 materials, the toggle cause incredible unstable crash to desktop touching anything in the interface kind of instability. I would also get this error before a hard crash:

![WindowsTerminal_OFuQyNlPUp](https://github.com/user-attachments/assets/91c46190-7d04-4372-b0d4-20bb56c650a6)

@Blenux looks like we might need to pull this one or polish? 

So for this release, I won't toggle this on by default and not mention it for now till 4.4.

**Comment by Blenux** [comment]
hmm interesting, will have a look later today, might just be a recent merge change that causing it.

**Comment by Blenux** [comment]
Been trying to reproduce this, but can't, what kind of materials were you using?

**Comment by Draise14** [comment]
Might be due to only being EEVEE materials, not plugged to cycles outputs? These are rendered in Cycles right?

**Comment by Blenux** [comment]
Yeah those previews are done with cycles in the background I believe.

**Comment by Draise14** [comment]
I guess 300 non-cycles materials might have been the cause of the crash. I will close.
